### ISSUE TITLE
[BUG] Liquid connector on non-fiat quote asset

### ISSUE THREAD

**Issue by fengkiej** [CoinAlpha/hummingbot#1331]
**Describe the bug**
Liquid connector will stuck in `Started tracking order book for { trading_pair }` when running `pure_market_making` on pair with non-fiat quote asset. I believe the problem is due to wrong rule parsing in `liquid_market.pyx` for https://api.liquid.com/products. Notice that in https://api.liquid.com/currencies, it has `minimum_order_quantity` field and when the program parses the data, it takes the quote currency's `minimum_order_quantity` making `quantize_order_amount` function always return 0.

**Steps To Reproduce**
Here are the steps to reproduce the issue (see attachments in the section below):
1. Set up your configuration file with these parameters: trading_pair = BTC-DAI or BTC-IDRT
2. Start the bot using this command: `start`

**Screenshots**
https://imgur.com/wx7Nk6h

**Release version**
0.22.0

**Attachments**
N/A

Discord: Fengkie | Rupiah Token#2430
ETH address: 0x8264CF2C91824c28EF7572348C3386Cb72199E0e

**Comment by dennisocana** [comment]
@fengkiej thanks for reporting this to us. Please send the log file as well when you encountered this error so we can further investigate.

**Comment by dennisocana** [comment]
@fengkiej thanks for reporting this to us. One of our support team ran some tests (but in paper trade) and was able to get it working (placing buy and sell orders) on BTC-IDRT in the same version.

Please send the **log file** when you encountered this error and your **config file** so we can further investigate.

**Comment by Hashira-Lars** [comment]
Tested and was able to get it working but not sure when it's live trade

btc/idrt
![image](https://user-images.githubusercontent.com/55869559/72771175-0736f280-3c3b-11ea-82e9-8329e499a858.png)

![image](https://user-images.githubusercontent.com/55869559/72771188-11f18780-3c3b-11ea-84bc-029263df7a32.png)

btc/dai
![image](https://user-images.githubusercontent.com/55869559/72771282-6ac12000-3c3b-11ea-8a61-a1948760ddfa.png)

![image](https://user-images.githubusercontent.com/55869559/72771305-79a7d280-3c3b-11ea-805f-d20bfb30a005.png)

**Comment by fengkiej** [comment]
@dennisocana @Hashira-Lars, thanks for responding. I just updated to 0.23.0 on dev branch and run pmm again with @Hashira-Lars's config live on BTC-IDRT. 

It's showing this error 
![Screenshot from 2020-01-21 22 17 45](https://user-images.githubusercontent.com/14020439/72817403-49445080-3c9c-11ea-9621-f09db9e4c342.png)

[conf_pure_market_making_strategy_0.txt](https://github.com/CoinAlpha/hummingbot/files/4091722/conf_pure_market_making_strategy_0.txt)

[logs_conf_pure_market_making_strategy_0.log](https://github.com/CoinAlpha/hummingbot/files/4091726/logs_conf_pure_market_making_strategy_0.log)

So, I reconfigured the order_amount to 0.01 and 0.1 but it is still showing the same error.
![Screenshot from 2020-01-21 22 32 00](https://user-images.githubusercontent.com/14020439/72818842-ba850300-3c9e-11ea-9ed8-eeec52a24073.png)

[conf_pure_market_making_strategy_0.txt](https://github.com/CoinAlpha/hummingbot/files/4091810/conf_pure_market_making_strategy_0.txt)

[logs_conf_pure_market_making_strategy_0.log]
(https://github.com/CoinAlpha/hummingbot/files/4091805/logs_conf_pure_market_making_strategy_0.log)

Then, I modified the source to make trading rules visible by adding the following snippet in `liquid_market.pyx`

```
    @property
    def trading_rules(self):
        return self._trading_rules
```

And enabled `debug_console`, as in the image, it is showing `min_order_size=500`.
![Screenshot from 2020-01-21 22 42 52](https://user-images.githubusercontent.com/14020439/72819582-1734ed80-3ca0-11ea-92be-dc538bd29638.png)

Referencing to 
```
    cdef object c_quantize_order_amount(self, str trading_pair, object amount, object price=s_decimal_0):
        """
        *required
        Check current order amount against trading rule, and correct any rule violations
        :return: Valid order amount in Decimal format
        """
        cdef:
            TradingRule trading_rule = self._trading_rules[trading_pair]

        global s_decimal_0
        quantized_amount = MarketBase.c_quantize_order_amount(self, trading_pair, amount)

        # Check against min_order_size. If not passing either check, return 0.
        if quantized_amount < trading_rule.min_order_size:
            return s_decimal_0

        # Check against max_order_size. If not passing either check, return 0.
        if quantized_amount > trading_rule.max_order_size:
            return s_decimal_0

        return quantized_amount
``` 
which is called through `quantize_order_amount(trading_pair, amount)` in `execute_buy` and `execute_sell`. The wrong parsing makes config's amount i.e. 0.1, returning 0 when  `quantize_order_amount(trading_pair, amount)` is called.

**Comment by fengkiej** [comment]
merged in #1332

**Comment by moraliang** [comment]
@fengkiej  Sent you 0.1ETH bug bounty.
